velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -773,8 +773,21 @@ void AggregationFuzzer::go() {
           // independent for comparison.
           auto mitigation = customVerificationFunctions_.at(signature.name);
           if (!mitigation.empty()) {
-            projections = groupingKeys;
-            projections.push_back(fmt::format(fmt::runtime(mitigation), "a0"));
+            bool ignoreMitigation = false;
+            if (mitigation.find("array_sort") != std::string::npos) {
```

Comment:
Why are you hard-coding array_sort here?

Reply:
Order-dependent functions depend on `array_sort` to sort the results, and they may fuzz `MapType` inputs, which may be the input of `array_sort`. 

https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/AggregationFuzzerTest.cpp#L64-L94
```C++
// Functions whose results verification should be skipped. These can be
  // order-dependent functions whose results depend on the order of input rows,
  // or functions that return complex-typed results containing floating-point
  // fields. For some functions, the result can be transformed to a value that
  // can be verified. If such transformation exists, it can be specified to be
  // used for results verification. If no transformation is specified, results
  // are not verified.
  static const std::unordered_map<std::string, std::string>
      customVerificationFunctions = {
       ...
          {"set_agg", "array_sort({})"},
          {"set_union", "array_sort({})"},
          {"map_agg", "array_sort(map_keys({}))"},
          {"map_union", "array_sort(map_keys({}))"},
          {"map_union_sum", "array_sort(map_keys({}))"},
```

https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/utils/AggregationFuzzer.cpp#L771-L789
```C++
if (customVerification) {
          // Add optional projection on the original result to make it order
          // independent for comparison.
          auto mitigation = customVerificationFunctions_.at(signature.name);
          if (!mitigation.empty()) {
            projections = groupingKeys;
            projections.push_back(fmt::format(fmt::runtime(mitigation), "a0"));
          }
        }

        auto input = generateInputData(argNames, argTypes);

        verifyAggregation(
            groupingKeys,
            {call},
            masks,
            input,
            customVerification,
            projections);
```

Reply:
@duanmeng That's a good point, but it is separate from having fuzzer respect 'orderable' property in function signatures. Let's create a GitHub issue to discuss how to address this issue.

Reply:
Got it, thanks.

Reply:
> @duanmeng That's a good point, but it is separate from having fuzzer respect 'orderable' property in function signatures. Let's create a GitHub issue to discuss how to address this issue.

@mbasmanova I've filed an issue https://github.com/facebookincubator/velox/issues/6999.